default/index.js

```diff
@@ -61,16 +58,36 @@ function readDirectory(dirName, stream) {
 
 // If stream is true, a stream is provided instead of a buffer for
 // each file object along with the corresponding size
-function getDefaultProjects(stream) {
-  var DefaultProjects = {};
+function cacheProjectFiles() {
   var projects = fs.readdirSync(__dirname);
```

Comment:
This should be fine, since it's for fetching static files that are committed into the tree.